### ISSUE TITLE
frontend: overview of schemas

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#52]
## User story
1. As a {}
2. I want / need {}
3. So that {}

## Acceptance criteria
* Criterion 1
* Criterion 2
* ...

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by veitmo** [comment]
@annikakrause Habe die Functionalities ergänzt. Auch hier gerne melden, falls etwas deiner Meinung nach noch nicht passt

**Comment by Idontker** [comment]
Done in sprint 5 https://github.com/amosproj/amos2022ss04-digital-identity/commit/8b9cdf974b0592bb76a4d87a3d35fe9834c38272
